nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -227,17 +227,17 @@ getMergeableGraph(std::shared_ptr<const GraphRepresentation> graph,
 
   /** FIXME :the layers is not the actual model_graph. It is just the vector of
    * layers generated by Model Loader. so graph[0] is still valid. Also we need
-   * to consider that the first layer of ini might be th first of layers. Need
+   * to consider that the first layer of ini might be the first of layers. Need
    * to change by compiling the backbone before using here. */
   std::string input_shape =
     iniparser_getstring(ini, (sec_name + ":Input_Shape").c_str(), "");
-  if (!input_shape.empty()) {
+  if (!input_shape.empty() && !g.empty()) {
```

Comment:
redundant check? already checked at line 195.

Reply:
Yes, it is redundant, updated.